application/cypress/integration/card/card.spec.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jest/expect-expect */
+/* eslint-disable jest/valid-expect */
+/// <reference types="cypress" />
+import { registrationHelper } from '../utils/registrationHelper';
+
+describe('Card should be checked', () => {
+  it('card reducer should be empty', () => {
+    registrationHelper();
+    cy.get('[data-cy=data-cy-app-cars]').click();
+    cy.get('[data-cy=data-cy-app-BMW]').click();
```

Comment:
Is it exact car item or category of cars?

Reply:
I gave more readable names for data attribute